spec/reducers/config/nodes/staticNodes.spec.ts

```diff
@@ -0,0 +1,128 @@
+import { web3SetNode, web3UnsetNode } from 'actions/config';
+import { staticNodes, INITIAL_STATE } from 'reducers/config/nodes/staticNodes';
+import { EtherscanNode, InfuraNode, RPCNode } from 'libs/nodes';
+import { Web3NodeConfig } from 'types/node';
+import { networkIdToName } from 'libs/values';
+import { Web3Service } from 'libs/nodes/web3';
+
+const expectedInitialState = JSON.stringify({
```

Comment:
Hm, don't really like that all this is duplicated. Is there really no way to import an object from `reducer` that is used to create the initial state?

```
const nodes = {..}

INITIAL_STATE =  {
    nodes,
    blah,

}
```

Reply:
I hard coded the expected initial state here instead of importing the initial state so that the test checks for equality, or else it doesnt do anything.

Reply:
We could just get rid of the initial state test too if you think thats too much overhead